QueryBuilder/Clauses/WhereClause.cs

```diff
@@ -24,7 +24,7 @@ public override string ToString()
         {
             var sql = string.Join($" {And} ", Conditions);
 
```

Comment:
Probably a nitpick, but I think it would be better to check `Conditions.Any()` rather than the output string `sql` here. Something like:

```csharp
if (!Conditions.Any())
{
    return string.Empty;
}

var sql = string.Join($" {And} ", Conditions);
return $"{Where} {sql}";
```

Reply:
You can use the `Any()` in a `IList`, but you can do as following.

```csharp
if (Conditions.Count == 0)
{
    return string.Empty;
}

var sql = string.Join($" {And} ", Conditions);
return $"{Where} {sql}";
```

Reply:
`Any()` is available on an `IEnumerable` as well.